packages/modules/src/lib/replaceArticleCount.tsx

```diff
@@ -32,6 +36,7 @@ export const replaceArticleCountWithLink = (
             dangerouslySetInnerHTML={{
                 __html: parts[parts.length - 1] as string,
             }}
+            key={key++}
```

Comment:
React was complaining in dev about these list elements not having keys, seemed the simplest way to solve.